geoxarray/accessor.py

```diff
@@ -78,23 +135,25 @@ def crs(self):
         coords_crs = self._obj.coords.get('crs')
         attrs_crs = self._obj.attrs.get('crs')
         area = self._obj.attrs.get('area')
-        cf_mapping_params = self.find_cf_grid_mapping()
-        self._is_gridded = True
-        if cf_mapping_params:
-            # TODO: Convert CF params to CRS object
-            self._cf_grid_mapping_parameters = cf_mapping_params
+        grid_mapping = self._obj.attrs.get('grid_mapping')
+        # TODO: Check for lon/lat 2D coordinate arrays
+        if grid_mapping:
+            self._crs = CRS.from_cf(self._obj[grid_mapping].attrs)
         elif coords_crs is not None:
-            crs = coords_crs
-            # TODO: Convert to geoxarray CRS object
+            crs = CRS.from_user_input(coords_crs)
         elif attrs_crs is not None:
-            crs = attrs_crs
-            # TODO: Convert to geoxarray CRS object
+            crs = CRS.from_user_input(attrs_crs)
         elif has_pyresample and isinstance(area, AreaDefinition):
-            # TODO: Convert and gather information from definitions
-            pass
+            if hasattr(area, 'crs'):
+                crs = area.crs
+            else:
+                crs = CRS.from_dict(area.proj_dict)
         elif has_pyresample and isinstance(area, SwathDefinition):
             # TODO: Convert and gather information from definitions
             self._is_gridded = False
+        else:
+            crs = False
```

Comment:
I think if you are going to set it to False, you will need to check for False in the statement at the top (not included in the diff) and return None it it is False.